chipy_org/apps/meetings/models.py

```diff
@@ -257,6 +257,8 @@ class RSVP(CommonModel):
 
     class Meta:
         ordering = ["-meeting", "last_name", "first_name"]
+        verbose_name = "RSVP"
+        verbose_name_plural = "RSPVs"
```

Comment:
Please change line 251 to `RSVPs`. Thanks!

Reply:
Actually, you can take out line 251 because by default, Django adds an 's' for the plural form of verbose name if none is specified.

Reply:
Dang, typing is hard.  I've made the update.  Thanks!